### ISSUE TITLE
Does ember-intl support Ember 2.x.x?

### ISSUE THREAD

**Issue by anotheredward** [yahoo/ember-intl#270]
The README.md reads:  > 1.1.3

After creating a brand new project, installing ember-intl, and adding "this.intl.set..." to app.js I get the following error message when visiting the index page:

```
TypeError: _ember.default.HTMLBars._registerHelper is not a function
    at registerIntl (ember-intl.js:23)
    at Object.initialize (ember-intl.js:50)
    at ember.debug.js:4467
    at ember.debug.js:4498
    at Object.visit [as default] (ember.debug.js:54796)
    at DAG.topsort (ember.debug.js:2325)
    at _emberRuntimeSystemNamespace.default.extend._runInitializer (ember.debug.js:4497)
    at _emberRuntimeSystemNamespace.default.extend.runInitializers (ember.debug.js:4458)
    at _emberRuntimeSystemNamespace.default.extend._bootSync (ember.debug.js:4359)
    at _emberRuntimeSystemNamespace.default.extend.domReady (ember.debug.js:4260)
```

Thanks :)

**Comment by clekstro** [comment]
I was able to install earlier this evening. Have you tried restarting your server?

**Comment by anotheredward** [comment]
I have restarted the server.
Using Node 4.2.1, Ember Cli 1.13.13, npm 2.14.10, on OSX
I took the following steps:
* `ember new intl-test`
* change the ember version from 1.13.11 to 2.2.0
* run bower install to update ember
* When prompted with "Unable to find a suitable version for ember, please choose one:" I chose " 3) ember#2.2.0 which resolved to 2.2.0 and is required by intl-test2"
* ember install ember-intl
* start the application
* the error message is displayed

**Comment by jasonmit** [comment]
`HTMLBars._registerHelper` isn't used anywhere in 2.0-rc-8, so I'm pretty sure you're on stable, ^1.0, which the documentation on master doesn't cover.

`ember install ember-intl@beta` should resolve your issue.  I'll likely be releasing 2.0-stable once I have some cycles to remove the documentation from formatjs.io.
